legislation_analysis/processing/pos_tagger.py

```diff
@@ -118,17 +121,20 @@ def process(
 
         # extract all part of speach elements
         for col in cols_to_apply_pos:
-            logging.debug(f"\tApplying POS tagging to {col[0]}...")
+            logging.debug(f"\tApplying POS tagging to {col[0]}.")
             self.pos_df[col[1]] = self.pos_df[col[0]].apply(self.pos_tag)
 
         # isolate parts of speech of interest
         if tags_of_interest:
-            logging.debug("\tExtracting tags of interest...")
+            logging.debug("\tExtracting tags of interest.")
 
             for col in cols_to_apply_pos:
-                logging.debug(f"\tExtracting tags of interest from {col[1]}...")
+                logging.debug(f"\tExtracting tags of interest from {col[1]}.")
                 col_name = f"{col[1]}_tags_of_interest"
 
                 self.pos_df[col_name] = self.pos_df[col[1]].apply(
                     lambda x: self.extract_tags_of_interest(x, tags_of_interest)
                 )
+                self.pos_df["joined_" + col_name] = self.pos_df[col_name].apply(
+                    self._join_numpy_array
+                )
```

Comment:
To properly do the `fit_transform` it needs to be in a string form, as opposed to an `ndarray`.